src/forms.ts

```diff
@@ -30,8 +36,12 @@ export type FieldErrorCode = keyof typeof FieldErrorCodeEnum;
 
 export interface FormError {
   field?: string;
-  code?: FormErrorCode | FieldErrorCode;
+  // @TODO: provide Stripe error codes from Formspree API so we can change the string type here to StripeErrorType
+  code?: FormErrorCode | FieldErrorCode | string;
```

Comment:
I don't think we need the `| string` anymore if we're now using the details block.

Reply:
Yup correct